### PR TITLE
confluent-cli: Add version 3.28.0

### PR DESCRIPTION
**Pull request by ogerardin**
- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/confluent-cli.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            "hash": "d6cb5ac8e08632ffdebc8344aa168b9492f478f9ce5cad0e6aebd91f1ecc0c41"
+        }
+    },
+    "bin": "confluent\\confluent.exe",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/confluentinc/cli/releases/download/v$version/confluent_$version_windows_amd64.zip"
+            }
+        }
```
[the remark this answers is not in the document]
Comment (ogerardin) [reply bucket/confluent-cli.json:24]: > Please add link to checksums file as well.

Thanks, this seems to work but where is it documented? I looked for checksums and didn't find them...

[the remark this answers is not in the document]
Comment (aliesbelik) [reply bucket/confluent-cli.json:24]: Here is something quite helpful:
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#adding-hash-to-autoupdate
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#properties-of-autoupdatehash

[the remark this answers is not in the document]
Comment (aliesbelik) [reply bucket/confluent-cli.json:24]: > Thanks, but what I don't get is where does this file $baseurl/confluent_$version_checksums.txt come from? it's not advertised on the release page... https://github.com/confluentinc/cli/releases/tag/v3.28.0
> Is it something that all githut releases have?

Just `confluent_3.28.0_checksums.txt` (https://github.com/confluentinc/cli/releases/download/v3.28.0/confluent_3.28.0_checksums.txt) among other files in release, as you can see.
`$baseurl` and `$version` are scoop [variables](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#url-variables).

[the remark this answers is not in the document]
Comment (ogerardin) [reply bucket/confluent-cli.json:24]: OK I must be blind, I hadn't seen the file in the middle of the other ones...

**Pull request merged**